app/views/content/steps-to-become-a-teacher/_step_6_start_your_teacher_training.erb

```diff
@@ -0,0 +1,7 @@
+<p>All postgraduate training involves classroom placements in at least 2 schools, with some theoretical learning – this might be in a different location to your placements.</p>
+
+<p>Most postgraduate courses start in September, with full-time courses taking 9 months and part-time courses taking 18 to 24.</p>
+
+<p>Undergraduate courses usually take 4 years.</p>
+
+<p><a href="/train-to-be-a-teacher/initial-teacher-training">Find out more about what teacher training is like</a>.</p>
```

Comment:
@EllieNodder Do we need to make it clear that this guidance is about **postgraduate** teacher training?